styles/paste.rs/catppuccin.user.css

```diff
@@ -61,7 +61,7 @@
     @accent-color: @catppuccin[@@lookup][@@accent];
```

Comment:
Huh looks like something is wrong with the accent-color.  I must have broken it in #557? I can't see what is wrong but elements like `a` aren't getting the right accent color applied, just the string name of the color (e.g. `mauve` instead of `#xxxxxx`).